src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Columns/action.html.twig

```diff
@@ -28,14 +28,29 @@
 {% if actions is not empty %}
   <div class="btn-group-action text-right">
     <div class="btn-group">
-      {{ include('@PrestaShop/Admin/Common/Grid/Actions/Row/'~actions|first.type~'.html.twig', {
-        'grid': grid,
-        'column': column,
-        'record': record,
-        'action': actions|first
-      }) }}
+      {% set skipedActions, isFirstRendered = 0, false %}
 
-      {% if actions|length > 1 %}
+      {# Render first item that is not in dropwdown #}
+      {% for action in actions %}
+        {% if not isFirstRendered %}
+          {% set skipedActions = skipedActions + 1 %}
+        {% endif %}
+
+        {% if action.isApplicable(record) and not isFirstRendered %}
```

Comment:
why do you need to ignore the first action?

Reply:
it has "special" rendering, first action is rendered differently, see example here https://prnt.sc/kqh91s

Reply:
it specific to every grid in BO.

Reply:
ok then :+1: